index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Expenser</title>
```

Comment:
Nedostaje nam
```suggestion
		<meta charset="UTF-8">
```
kako bi mogli da handle-ujemo bez problema cirilicu i specijalne karaktere na primer